test/lit.cfg

```diff
@@ -53,7 +53,9 @@ def darwin_sdk_build_version_split(version):
 
 
 def darwin_sdk_build_version_cmp(lhs, rhs):
-    return cmp(
+    def _cmp(a, b):
```

Comment:
I would say it's worth commenting that this is for Python 3 compatibility, otherwise someone will come later and replace it back.